components/server/src/ome/services/sharing/ShareBean.java

```diff
@@ -752,6 +758,35 @@ public void invalidateConnection(long shareId, Experimenter exp) {
         throw new UnsupportedOperationException();
     }
 
+    @RolesAllowed("user")
+    @Transactional(readOnly = false)
+    public void notifyMembersOfShare(long shareId, String subject, String message,
+            boolean html, List<Experimenter> exps) {
+
+        String sender = mailUtil.getSender();
+        if (StringUtils.isBlank(sender)) {
+            log.error("omero.mail.from cannot be empty.");
+            return;
+        }
+
+        Set<Long> memberIds = new HashSet<Long>();
+        for (final Experimenter e : getAllMembers(shareId)) {
+            memberIds.add(e.getId());
+        }
+        for (final Experimenter e : exps) {
+            if (memberIds.contains(e.getId())){
+                if (e.getEmail() != null && mailUtil.validateEmail(e.getEmail())) {
+                    try {
+                        mailUtil.sendEmail(e.getEmail(), subject, message, html,
+                                null, null);
+                    } catch (MailException me) {
+                        log.error(me.getMessage());
```

Comment:
I will ignore it, as when share is created page is redirected anyway if new comment is added it just update html content. I think there is no space for any messages.